tests/e2e/standalone/init_test.go

```diff
@@ -110,6 +127,26 @@ func TestStandaloneInit(t *testing.T) {
 		verifyBinaries(t, daprPath, daprRuntimeVersion, daprDashboardVersion)
 		verifyConfigs(t, daprPath)
 	})
+
+	t.Run("init without runtime-version flag", func(t *testing.T) {
+		// Ensure a clean environment
+		must(t, cmdUninstall, "failed to uninstall Dapr")
+
+		output, err := cmdInit()
+		t.Log(output)
+		require.NoError(t, err, "init failed")
+		assert.Contains(t, output, "Success! Dapr is up and running.")
+
+		homeDir, err := os.UserHomeDir()
+		require.NoError(t, err, "failed to get user home directory")
+
+		daprPath := filepath.Join(homeDir, ".dapr")
+		require.DirExists(t, daprPath, "Directory %s does not exist", daprPath)
+
+		verifyContainers(t, daprRuntimeVersion)
+		verifyBinaries(t, daprPath, daprRuntimeVersion, daprDashboardVersion)
+		verifyConfigs(t, daprPath)
```

Comment:
This will fail whenever there is a new patch release, unless the DAPR_RUNTIME_VERSION field in workflow file is updated ...

Reply:
Don't we query the latest version in the workflow file though, with the cURL magic?

Reply:
not for normal scheduled runs ... only for PRs... That is done specifically such that for release branches we always have a specific version to target and not latest version ...

Reply:
Got it, yes. Then we should check "greater-than-equal" for versions in the test, WDYT?

Reply:
`init` as of now is dependent on latest release in GH. There are two scenarios here: 
- without specifying a runtime version, it should always get the latest version. we can follow the `cURL` magic and injection into env there? 
- when runtime version is specified, it has to be done in workflow file and that version should be checked. 
IMO we should not relax the test assertion.

Reply:
Just to clarify, the "greater-than-equal" was only for `init` without runtime version.

Yes, in the workflow, we can set additional variables to always point to the latest version. The scheduled runs can set the variables, but still be pinned to the release specified.

Reply:
I see, let me check this.